python/src/aiconfig/editor/client/src/components/prompt/PromptMenuButton.tsx

```diff
@@ -31,10 +46,19 @@ export default memo(function PromptMenuButton({
       </Menu.Target>
 
       <Menu.Dropdown>
+        {prompt.outputs?.length ? (
+          <Menu.Item
+            icon={<IconEraser size={16} />}
+            color="white"
+            onClick={deleteOutput}
+          >
```

Comment:
cc @rholinshead for whether this would work with light mode theming

Reply:
Good catch. I think we can remove color here and it will have the correct color we want. To test this, you can set `mode="gradio"` and `themeMode="dark"` and `themeMode="light"` props for AIConfigEditor in the `LocalEditor` component